File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 1:
            return len(nums)
        i = 0
        while i < len(nums)-1:
            if nums[i] == nums[i+1]:
                nums.pop(i)
                continue
            i += 1
        
        return len(nums)

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
#include <unordered_map>
#include <vector>
#include <iostream>

using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> m;

        int i = 0;
        for(; i < nums.size(); i++){
            m[nums[i]] = i;
        }

        for(i = 0; i < nums.size(); i++){
            int difference = target - nums[i];
            if(m.count(difference) && m[difference] != i){
                return {i, m[difference]};
            }
        }
        return {};
    }
};